client/my-sites/theme/main.jsx

```diff
@@ -728,9 +727,9 @@ class ThemeSheet extends Component {
 		if ( hasWpComThemeUpsellBanner ) {
 			pageUpsellBanner = (
 				<UpsellNudge
-					plan={ PLAN_PREMIUM }
+					plan={ PLAN_WPCOM_PRO }
```

Comment:
This has created a path for a legacy plan to transit to the Pro plan, but we will leave it for now: p2-pdgrnI-xR#comment-1052